samila/functions.py

```diff
@@ -233,6 +233,19 @@ def generate_params_filter(
     g.seed, g.start, g.step, g.stop = seed, start, step, stop
 
 
+def save_image_filter(g, depth=None):
```

Comment:
My suggestion:

Change this function name to `save_params_filter` or `save_image_params_filter`